platform/os_windows.go

```diff
@@ -343,3 +373,33 @@ func ReplaceFile(source, destination string) error {
 
 	return windows.MoveFileEx(src, dest, windows.MOVEFILE_REPLACE_EXISTING|windows.MOVEFILE_WRITE_THROUGH)
 }
+
+/*
+	Sample output:  Get-NetAdapter | Select-Object MacAddress, PnpDeviceID| Format-Table -HideTableHeaders
+
+6C-A1-00-50-E4-2D PCI\VEN_8086&DEV_2723&SUBSYS_00808086&REV_1A\4&328243d9&0&00E0
+80-6D-97-1E-CF-4E USB\VID_17EF&PID_A359\3010019E3
+*/
+func FetchMacAddressPnpIDMapping(execClient ExecClient) (map[string]string, error) {
+	ctx, cancel := context.WithTimeout(context.Background(), 5*time.Second)
```

Comment:
was the Timeout const in cns util supposed to be in this package?

Reply:
Addressed